backend/src/views/feed.pug

```diff
@@ -1,12 +1,25 @@
 extends layout
 
 block content
-  h1 Your feed
-  ul
-  each mnemo in mnemos
-    li
-      h3
-        a(href='/mnemo?title=' + mnemo.title) #{mnemo.title}
-      p= mnemo.place
-      p= mnemo.text
-      p= mnemo.date.day + "." + mnemo.date.month + "." + mnemo.date.year
+  main
+    h1 Your feed
+    p Mmemo search:
+    input.search(
+      type='text'
+      name='search'
+      id='searchInput'
+      placeholder='Enter a search term'
+    )
+    button(onclick='searchMnemo()') Search
+    script.
+      function searchPlace() {
+        window.open(`http://localhost:3000/mnemo?title=${document.getElementById('searchInput').value}`, '_self')
```

Comment:
you should use a <form> here, and instead of implementing a window.open, you should use the form action and url and render the results page.